regression/pages/whitelabel/home_page.py

```diff
@@ -107,3 +107,9 @@ def go_to_registration_page(self):
         Go to registration page
         """
         self.q(css='.btn-brand.btn-client[href="/register"]').click()
+
+    def go_to_courses_page(self):
+        """
+        Go to courses page
+        """
+        self.q(css='.brand-link[href="/courses"]').click()
```

Comment:
My concern is that there might be other tests written in the future that use this method, and they could be flaky if the author doesn't know or remember to put a wait_for_page in **their** test. 
I'm now realizing that this action changes what page you're on. 
So maybe the best way to mitigate that risk is to add a comment here reminding people that if they use this method, then they need a wait_for_page statement in their test right after it.